docs/DeveloperGuide.md

```diff
@@ -581,6 +582,21 @@ Use case ends.
 
 Use case ends.
 
+**Use case: UC07 Undo the last command**
+
+**MSS**
+
+1. User requests to undo the last command.
+2. McGymmy successfully undo the last command.
+
+Use case ends.
+
+**Extensions**
+
+1a. McGymmy notices that there is no command left to undo.
```

Comment:
Might want to add that McGymmy throws an error

Reply:
I've include it as an extension already, i think?

Reply:
I think we shouldn't state specifically that mcgymmy throws an error. We should keep our use cases as general as possible

Reply:
Hmm i think notices is a little to vague though. It can range from logging it down and not showing any errors to like crashing outright

Reply:
Updated